CONTRIBUTING.md

```diff
@@ -27,9 +27,14 @@ If you are reporting a security bug or vulnerability, see SECURITY.md.
 
 ## Developer Certificate of Origin
 
-We respect the intellectual property rights of others and we want to make sure all incoming contributions are correctly attributed and licensed. A Developer Certificate of Origin (DCO) is a lightweight mechanism to do that. The DCO is a declaration attached to every commit. In the commit message of the contribution, the developer simply adds a `Signed-off-by` statement and thereby agrees to the DCO, which you can find below or at [DeveloperCertificate.org](http://developercertificate.org/).
+We respect the intellectual property rights of others and we want to make sure
+all incoming contributions are correctly attributed and licensed. A Developer
+Certificate of Origin (DCO) is a lightweight mechanism to do that. The DCO is
+a declaration attached to every commit. In the commit message of the contribution,
+the developer simply adds a `Signed-off-by` statement and thereby agrees to the DCO,
+which you can find below or at [DeveloperCertificate.org](http://developercertificate.org/).
```

Comment:
Moving linebreaks to beween sentences and after comma in some cases. This linebreak has no effect on the rendered output on the web page. It's per the valkey-doc style guidelines.

```suggestion
all incoming contributions are correctly attributed and licensed.
A Developer Certificate of Origin (DCO) is a lightweight mechanism to do that.
The DCO is a declaration attached to every commit.
In the commit message of the contribution,
the developer simply adds a `Signed-off-by` statement and thereby agrees to the DCO,
which you can find below or at [DeveloperCertificate.org](http://developercertificate.org/).
```